### ISSUE TITLE
Docker on Unraid

### ISSUE THREAD

**Issue by PureLoneWolf** [GregYankovoy/docker-chowdown#1]
Hi there

Is there a way to change the location of the stored recipes/components?  I run the Docker on Unraid and don't want to have to expose my appdata share and edit everything there.

Many thanks

**Comment by GregYankovoy** [comment]
You should be able to create an additional additional volume for the recipes and/or component folders. (example for _recipes in below screenshot).  Keep in mind that the container will need to be running under a PUID/PGID with access to the files, and that the container may not auto-copy the default recipes.

![image](https://user-images.githubusercontent.com/34084447/53295807-76a92580-37d1-11e9-8e7c-7f7b883e29ba.png)

Let me know if this works. If not, we can talk about how to get around the issue or update the container to support this use case.

**Comment by PureLoneWolf** [comment]
Superb, that worked.  I had tried that, but didn't specify /config, just /_recipes

So I have now got a "Food" share on unraid with folders for components, images and recipes.  I am not worried about the default recipes as I will be using my own.  Now to see which app I prefer to use to access them :)

![image](https://user-images.githubusercontent.com/5523493/53299669-a2fa8b80-383d-11e9-9a88-78cb726d5b84.png)
